HRIS.Services/Services/EmployeeService.cs

```diff
@@ -122,45 +128,54 @@ public async Task<List<EmployeeDto>> GetAll(string userEmail = "")
 
     public async Task<EmployeeDto?> GetEmployee(string email)
```

Comment:
```suggestion
    public async Task<EmployeeDto?> GetEmployeeByEmail(string email)
```

Naming of methods is important to describe them better